git_machete/cmd.py

```diff
@@ -1065,16 +1065,17 @@ def print_new_line(new_status: bool) -> None:
 
     def status(self, warn_on_yellow_edges: bool) -> None:
         dfs_res = []
+        prefix: str
 
-        def prefix_dfs(u_: str, prefix: List[Optional[str]]) -> None:
-            dfs_res.append((u_, prefix))
+        def prefix_dfs(u_: str, prefix_df: List[Optional[str]]) -> None:
```

Comment:
Maybe rename `prefix_df` to `accumulated_path`, it will be more telling (since this param basically corresponds to a path in a tree &mdash; from a tree root to the current vertex)